client/state/sites/selectors.js

```diff
@@ -166,3 +166,34 @@ export function isCurrentSitePlan( state, siteId, planProductId ) {
 
 	return sitePlan.product_id === planProductId;
 }
+/**
+ * Get primary site
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Object}         Site
+ **/
+export function getPrimarySite( state ) {
+	// need to figure out best way to retrieve primary site
+	// since this attribute doesn't exist by default
+	return filter( state.sites.items, ( site ) => site.primary ).shift();
```

Comment:
I assumed that what `[][0]`  does also, but maybe only in chrome....
<img width="104" alt="zrzut ekranu 2016-06-20 o 18 51 24" src="https://cloud.githubusercontent.com/assets/3775068/16202910/166484a6-3718-11e6-8f61-c102b15acc52.png">

Thanks!